package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "sms-segments-calculator",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "test": "tsc && jest",
+    "build": "tsc",
+    "dev": "tsc -w",
+    "lint": "eslint --ext ts src/",
+    "lint:fix": "npm run lint -- --fix",
+    "webpack": "webpack --config webpack.config.js",
+    "release": "tsc && webpack --config webpack.config.js"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "grapheme-splitter": "^1.0.4",
+    "jest": "^27.0.6"
```

Comment:
I think Jest could be a devDependency instead of a dependency

Reply:
Great catch @HugoCrd ! Fixed in da3dfb0